.github/workflows/lint.yml

```
@@ -1,45 +1,52 @@
           pip install -r python/requirements-dev.txt
           make develop
-      - name: Linting with ruff
+
+      - name: Linting with Ruff
         run: |
           source ~/.venv/ci-venv/bin/activate
           cd python && ruff $(git ls-files '*.py')
-      - name: Static analysis with mypy
+
+      - name: Static analysis with MyPy
         run: |
```

actionlint

shellcheck reported issue in this script: SC1090:warning:1:8: ShellCheck can't follow non-constant source. Use a directive to specify location

(shellcheck)